metadata-ingestion/src/datahub/ingestion/source/bigquery_v2/bigquery.py

```diff
@@ -142,7 +142,7 @@ def cleanup(config: BigQueryV2Config) -> None:
 @support_status(SupportStatus.CERTIFIED)
 @capability(
     SourceCapability.PLATFORM_INSTANCE,
-    "Not supported since BigQuery project ids are globally unique",
+    "Configurable platform instance is still not supported. Platform instance is always added using project id value",
```

Comment:
The comment "still not supported" implies to me that it may be supported in the future.

```suggestion
    "Platform instance is pre-set to the BigQuery project id",
```